client/tx/client.go

```diff
@@ -0,0 +1,114 @@
+package tx
+
+import (
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	ojoapp "github.com/ojo-network/ojo/app"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+	tmjsonclient "github.com/tendermint/tendermint/rpc/jsonrpc/client"
+)
+
+const (
+	gasAdjustment = 1
+)
+
+type Client struct {
```

Comment:
```suggestion
// TxClient helps with submitting transactions, setting up a keyring, etc
type TxClient struct {
```

Reply:
https://github.com/ojo-network/ojo/pull/103/commits/ea33a8a5da3c076e1936cfbc919727bbe6a92a6e I went ahead and updated all the "QueryClient" and "TxClient" names